gini/gini-vision-lib-ios
# Uploading state in pages collection

Added new state view in pages collection, being possible now to see if a document has been uploaded or not.
Also added scrollview inside the main collection to make the image zoomable.

### How to test
Run the example app, take a picture and see if the state of the upload in the multipage review view controller.

### Merging
Automatic
